ec2/sc-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -388,6 +400,42 @@ Resources:
           Value: "yes"
         - Key: "PatchGroup"
           Value: "prod-default"
+
+  EC2TargetGroup:
+    Type: AWS::ElasticLoadBalancingV2::TargetGroup
+    Properties:
+      Name: !Sub 'TargetGroup-${LinuxInstance}'
+      HealthCheckIntervalSeconds: 30
+      HealthCheckProtocol: HTTP
```

Comment:
This type should actually be HTTPS or it will create SSL connect errors in the web server logs, and probably show up as healthcheck failed.

Reply:
Fixed